apps/web/pages/apps/installed/[category].tsx

```diff
@@ -112,37 +106,95 @@ interface IntegrationsListProps {
+      utils.viewer.getUserMetadata.invalidate();
+    },
+  });
+
+  const { t } = useLocale();
   return (
-    <List>
-      {data.items
-        .filter((item) => item.invalidCredentialIds)
-        .map((item) => (
-          <AppListCard
-            key={item.name}
-            description={item.description}
-            title={item.name}
-            logo={item.logo}
-            isDefault={item.isGlobal}
-            shouldHighlight
-            slug={item.slug}
-            invalidCredential={item.invalidCredentialIds.length > 0}
-            actions={
-              <div className="flex  justify-end">
-                <ConnectOrDisconnectIntegrationButton
-                  credentialIds={item.credentialIds}
-                  type={item.type}
-                  isGlobal={item.isGlobal}
-                  installed
-                  invalidCredentialIds={item.invalidCredentialIds}
-                  handleDisconnect={handleDisconnect}
-                />
-              </div>
-            }>
-            <AppSettings slug={item.slug} />
-          </AppListCard>
-        ))}
-    </List>
+    <>
+      <List>
+        {data.items
+          .filter((item) => item.invalidCredentialIds)
+          .map((item) => {
+            const appSlug = item?.slug;
+            const appIsDefault =
+              appSlug === usersMetadata?.defaultConferencingApp?.appSlug ||
+              (appSlug === "daily-video" && !usersMetadata?.defaultConferencingApp?.appSlug);
+            return (
+              <AppListCard
+                key={item.name}
+                description={item.description}
+                title={item.name}
+                logo={item.logo}
+                isDefault={appIsDefault}
+                shouldHighlight
+                slug={item.slug}
+                invalidCredential={item.invalidCredentialIds.length > 0}
+                actions={
+                  <div className="flex  justify-end">
+                    <Dropdown modal={false}>
+                      <DropdownMenuTrigger asChild>
+                        <Button StartIcon={FiMoreHorizontal} variant="icon" color="secondary" />
+                      </DropdownMenuTrigger>
+                      <DropdownMenuContent>
+                        {!appIsDefault && variant === "conferencing" && (
+                          <DropdownMenuItem>
+                            <DropdownItem
+                              type="button"
+                              color="secondary"
+                              StartIcon={FiVideo}
+                              onClick={() => {
+                                const locationType = getEventLocationTypeFromApp(
+                                  item?.locationOption?.value ?? ""
+                                );
+                                if (locationType?.linkType === "static") {
+                                  setLocationType({ ...locationType, slug: appSlug });
```

Comment:
If the location is a link type - we set the location type so we can display a modal